platform/windows/detect.py

```diff
@@ -180,37 +180,37 @@ def configure_msvc(env, manual_msvc_config):
 
     if env["target"] == "release":
         if env["optimize"] == "speed":  # optimize for speed (default)
-            env.Append(CCFLAGS=["/O2"])
+            env.Append(CCFLAGS=["/O2", "/MP"])
         else:  # optimize for size
-            env.Append(CCFLAGS=["/O1"])
+            env.Append(CCFLAGS=["/O1", "/MP"])
         env.Append(LINKFLAGS=["/ENTRY:mainCRTStartup"])
         env.Append(LINKFLAGS=["/OPT:REF"])
 
     elif env["target"] == "release_debug":
         if env["optimize"] == "speed":  # optimize for speed (default)
-            env.Append(CCFLAGS=["/O2"])
+            env.Append(CCFLAGS=["/O2", "/MP"])
         else:  # optimize for size
-            env.Append(CCFLAGS=["/O1"])
+            env.Append(CCFLAGS=["/O1", "/MP"])
         env.AppendUnique(CPPDEFINES=["DEBUG_ENABLED"])
         env.Append(LINKFLAGS=["/OPT:REF"])
 
     elif env["target"] == "debug":
-        env.AppendUnique(CCFLAGS=["/Z7", "/Od", "/EHsc"])
+        env.AppendUnique(CCFLAGS=["/Z7", "/Od", "/EHsc", "/MP"])
         env.AppendUnique(CPPDEFINES=["DEBUG_ENABLED"])
-        env.Append(LINKFLAGS=["/DEBUG"])
+        env.Append(LINKFLAGS=["/DEBUG:fastlink"])
 
     env.Append(LINKFLAGS=["/SUBSYSTEM:WINDOWS"])
 
     if env["debug_symbols"] == "full" or env["debug_symbols"] == "yes":
         env.AppendUnique(CCFLAGS=["/Z7"])
-        env.AppendUnique(LINKFLAGS=["/DEBUG"])
+        env.AppendUnique(LINKFLAGS=["/DEBUG:fastlink"])
```

Comment:
Not sure about this one, `debug_symbols` is used to add debug symbols to release/release_debug binaries, which we still want to optimize as well as we can. What's the impact of `:fastlink` on the resulting binary (size, performance)?